README.md

```diff
@@ -9,6 +9,20 @@
 The example plugin prints "example" on every query that go handled by the server. It serves as
 documentation for writing CoreDNS plugins.
 
+## Compilation
+
+This package will always be compiled as part of CoreDNS and not in a standalone way. It will require you to use ''go get'' or as a dependency on [plugin.cfg](https://github.com/coredns/coredns/blob/master/plugin.cfg).
+
+The [manual](https://coredns.io/manual/toc/#what-is-coredns) will have more information about how to configure and extend the server with external plugins.
+
+A simple way to consume this plugin, is by adding the following on [plugin.cfg](https://github.com/coredns/coredns/blob/master/plugin.cfg), and recompile it as defined on the [manual](https://coredns.io/manual/toc/#writing-plugins):
```

Comment:
~~~
and recompile it as [detailed on coredns.io](https://coredns.io/2017/07/25/compile-time-enabling-or-disabling-plugins/#build-with-compile-time-configuration-file).
~~~